src/mngt/ocf_mngt_cache.c

```diff
@@ -3157,9 +3183,8 @@ static void _ocf_mngt_cache_unplug(ocf_cache_t cache, bool stop,
 	__deinit_promotion_policy(cache);
 
 	if (!stop) {
-		/* Just set correct shutdown status */
-		ocf_metadata_set_shutdown_status(cache, ocf_metadata_detached,
-				_ocf_mngt_cache_unplug_complete, context);
+		/* Skip metadata update - will be zeroed later in detach pipeline */
+		ocf_pipeline_next(ctx->pipeline);
```

Comment:
You can use `OCF_PL_NEXT_RET()` here, so you can drop the `else` block.

Reply:
I've reverted this part according to Michal's comment

Reply:
...and done it again with OCF_PL_NEXT_RET ;)